docs/TektonPipeline.md

```diff
@@ -283,3 +292,28 @@ A high level descriptions are given here. To get the detailed information please
 > #### Note:
 > * `kube-api-qps` and `kube-api-burst` will be multiplied by 2 in pipelines controller. To get the detailed information visit [Performance Configuration](https://tekton.dev/docs/pipelines/tekton-controller-performance-configuration/) guide
 > * if you modify or remove any of the performance properties, `tekton-pipelines-controller` deployment and `config-leader-election` config-map (if `buckets` changed) will be updated, and `tekton-pipelines-controller` pods will be recreated
+
+### Options
+```yaml
+spec:
+  # omitted other fields ...
+  options:
+    disabled: false
+    configMaps: {}
+    deployments: {}
+    statefulsets: {}
+    horizontalPodAutoscalers: {}
+    webhookConfigurationOptions:
+      validation.webhook.pipeline.tekton.dev:
+        failurePolicy: Fail
+        timeoutSeconds: 20
+        SideEffects: None
+```
+Options fields are optionnal and there is no default values. If user passes them, operator will take them in account. A high level descriptions are given here 
+
+* `disable` - enable or disable additional options
+* `configMaps` - additional configmaps to be created by the operator
+* `deployments` - additional deployments to be created by the operator
+* `statefulsets` - additional statefulsets to be created by the operator
+* `horizontalPodAutoscalers` - additional horizontalPodAutoscalers to be created by the operator
+* `webhookConfigurationOptions` - additional options for pipelines webooks, To get detailed information about webhooks options visit https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
```

Comment:
@jkhelil can you move the documentation under,
https://github.com/tektoncd/operator/blob/main/docs/TektonConfig.md#additional-fields-as-options

https://tekton.dev/docs/operator/tektonconfig/#additional-fields-as-options

Reply:
done